tests/testTimeTransformations.py

```diff
@@ -0,0 +1,353 @@
+
+        UTC values are generated by entering calendar dates into the USNO
+        JD calculator at
+
+        http://aa.usno.navy.mil/data/docs/JulianDate.php
+
+        TAI values are found from this table listing the relationship between
+        TAI and UTC as a function of epoch
+
+        https://hpiers.obspm.fr/eop-pc/earthor/utc/TAI-UTC_tab.html
+        """
+
+        jd_to_mjd = 2400000.5
+        sec_to_day = 1.0/86400.0
+
+        cls.utc_control = []
+        cls.tai_control = []
+        cls.year = []
+
+        # all UTC values calculated at midnight UT
+        # June 10, 1961
+        mjd = 2437460.5 - jd_to_mjd
+        dt = 1.4228180 + (mjd-37300.0)*0.001296
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1961)
+
+        # June 10, 1962
+        mjd = 2437825.5 - jd_to_mjd
+        dt = 1.8458580 + (mjd-37665.0)*0.0011232
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1962)
+
+        # June 10, 1963
+        mjd = 2438190.5 - jd_to_mjd
+        dt = 1.8458580 + (mjd - 37665.0) * 0.0011232
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1963)
+
+        # June 10, 1964
+        mjd = 2438556.5 - jd_to_mjd
+        dt = 3.3401300 + (mjd - 38761.0)*0.001296
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1964)
+
+        # June 10, 1965
+        mjd = 2438921.5 - jd_to_mjd
+        dt = 3.6401300 + (mjd-38761.0)*0.001296
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1965)
+
+        # June 10, 1974
+        mjd = 2442208.5 - jd_to_mjd
+        dt = 13.0
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1974)
+
+        # June 10, 1984
+        mjd = 2445861.5 - jd_to_mjd
+        dt = 22.0
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1984)
+
+        # June 10, 1994
+        mjd = 2449513.5 - jd_to_mjd
+        dt = 28.0
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1994)
+
+        # June 10, 1997
+        mjd = 2450609.5 - jd_to_mjd
+        dt = 30.0
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(1997)
+
+        # June 10, 2003
+        mjd = 2452800.5 - jd_to_mjd
+        dt = 32.0
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(2003)
+
+        # June 10, 2013
+        mjd = 2456453.5 - jd_to_mjd
+        dt = 35.0
+        cls.utc_control.append(mjd)
+        cls.tai_control.append(mjd + dt*sec_to_day)
+        cls.year.append(2013)
+
```

Comment:
Can you add a test at or around 2015-07-01? I did a quick test matching UTC/TAI between ModifiedJulianDate and astropy, and find a 1^-5 difference at this date when setting up ModifiedJulianDate with 
>>> md = ModifiedJulianDate(TAI=57204.)
>>> ast = Time(57204, format='mjd', scale='tai')
>>> ast.utc.mjd
57203.999583338154
>>> md.UTC
57203.99959490741
>>> md.TAI
57204.0
>>> ast.tai.mjd
57204.0
>>> 

(this is probably when a leap second is occuring I think? the difference is ~1 second)